client/components/domains/domain-search-results/index.jsx

```diff
@@ -144,20 +137,29 @@ class DomainSearchResults extends React.Component {
 				offer = null;
 			}
 
+			const noticeMessage = translate(
+				'If you already own this domain, you can use it for your WordPress.com site.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'If you already own this domain you can use it for your WordPress.com site.' )` *ES Score: 8*